### DISCUSSION TITLE
Why does Prisma VsCode extension format my .tsx files?

### DISCUSSION THREAD

**Discussion by mrmanicou** [prisma/prisma#4122] [Q&A] [answered]
`export default function Home(props) {
  return <div>Hey</div>;
}`

That "props" argument is automatically added by the extension when I hit save. I disabled all my extensions and reenabled them one by one. This is how I was able to identify that the Prisma extension was the culprit. Is this a bug? I think so... and if not, how do I disable it?

**Comment by ryands17** [comment, marked as answer]
Hey @mrmanicou 👋 
Thanks for creating an issue https://github.com/prisma/prisma/issues/4123. We will look into this :)

**Reply by ryands17** [reply]
Hey @mrmanicou 
This is a known issue and we are looking into this. As a workaround, you can add this to VSCode settings:

```json
"prisma.plugin.nextjs.addTypesOnSave": false,
"prisma.plugin.nextjs.hasPrompted": true
```

**Reply by mrmanicou** [reply]
Thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
